osquery/core/watcher.cpp

```diff
@@ -128,8 +138,25 @@ CLI_FLAG(uint64,
 
 CLI_FLAG(bool, disable_watchdog, false, "Disable userland watchdog process");
 
+HIDDEN_FLAG(bool,
```

Comment:
Why hide it?

Reply:
Good question, no reason.

Changed to `CLI_FLAG`.